### ISSUE TITLE
GDrive error

### ISSUE THREAD

**Issue by arielmoraes** [ddurdle/GDrive-for-KODI#154]
Hello, I'm having trouble starting the add-on, when I try to run it it says GDrive Error Check the log, but the Kodi log and the add-on log show nothing. I even tried to run the add-on without an account but I'm still getting the error.

I have the last Kodi version.

**Comment by negroISO** [comment]
Same thing happens on a few, but not all of my Kodi machines. OpenElec with latest G-Drive it happens, then I click a file, and it says that 4 or 5 can't be played then randomly picks one. On one of my Shield TV 2017's with Kodi 17.4 I get the error about G-Drive but it works fine, it's just when I originally launch Kodi. Then finally on another Shield TV 2017, I get the message once in a few days and again it continues to work fine.

I have the logs from the one that's not working and attached them for ddurdle
[Archive.zip](https://github.com/ddurdle/GDrive-for-KODI/files/1279488/Archive.zip)

**Comment by ddurdle** [comment]
Are you watching using STRM files? 

Can you clarify what "4 or 5 can't be played then randomly picks one" means?

**Comment by negroISO** [comment]
Yes, trying to play locally stored STRM files. In the lock you see I tried
to play S09E01.mkv.strm and then it proceeds to say skipping unplayable
file . . . . then randomly places another file down the list. If you hit
stop, and click on S09E01 . . .it plays another file closer to it, if you
repeat this a few times you end up finally getting S09E01.

However, I removed the G-Drive plugin from the Addon menu, rebooted,
reinstalled and tried to add a new oath account and it would error out on
adding, then it went ahead and added but when you click the account it just
spins forever.

**Comment by ddurdle** [comment]
Do you have streamer turned on in Experimental tab in settings?

**Comment by negroISO** [comment]
I checked, and it was not. I also removed the /userdata/plugin.g-drive
folder as well as the /addon/plugin.g-drive folder as well to give a fresh
install of the plugin.
